blocks/init/components/google-rich-snippets/google-rich-snippets.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Google Rich Snippets
+ *
+ * @package Eightshift_Boilerplate\Views\Parts
+ *
+ * @since 1.0.0
+ *
+ * TODO: Create this as a class with attributes.
+ */
+
+use Eightshift_Boilerplate\Core\Config;
+use Eightshift_Libs\Manifest\Manifest;
+
+$logo_img = apply_filters( Config::get_config( Manifest::MANIFEST_ITEM_FILTER_NAME ), 'logo.svg' );
+?>
+
+<!-- Google Rich Snippets -->
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "NewsArticle",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://google.com/article"
+    },
+    "headline": "<?php the_title(); ?>",
+  <?php if ( ! empty( $image ) ) { ?>
+  "image": {
+    "@type": "ImageObject",
+    "url": "<?php echo esc_html( $image['image'] ); ?>",
+    "height": <?php echo esc_html( $image['height'] ); ?>,
+    "width": <?php echo esc_html( $image['width'] ); ?>
+  },
+  <?php } ?>
+  "datePublished": "<?php echo esc_html( get_the_time( 'c' ) ); ?>",
+  "dateModified": "<?php echo esc_html( date( 'c', strtotime( $post->post_modified ) ) ); ?>",
```

Comment:
These too should be escaped using `esc_attr`

Reply:
👍